packages/fxa-settings/src/components/DataCollection/index.tsx

```diff
@@ -72,21 +61,19 @@ export const DataCollection = () => {
           </div>
 
           <div className="flex-1 text-center tablet:pt-1">
-            <form>
-              <Switch
-                {...{
-                  isSubmitting,
-                  isOn: metricsOptOut,
-                  id: 'metrics-opt-out',
-                  handler: handleMetricsOptOutToggle,
-                  localizedLabel: (
-                    <Localized id="dc-subheader">
-                      Analytics and Improvements
-                    </Localized>
-                  ),
-                }}
-              />
-            </form>
+            <Switch
+              {...{
+                isSubmitting: account.loading,
```

Comment:
I just realized I picked up this passing in the entire object pattern over setting the individual props from Experimenter and we don't do that as much in Settings 😅

Will r+ after this `loading` bug is addressed. I think I may also change this prop to `disabled` in my PR since that's what we're using for other buttons.